actor-util/src/main/java/org/sunbird/actorutil/location/LocationUtil.java

```diff
@@ -0,0 +1,156 @@
+package org.sunbird.actorutil.location;
+
+import akka.actor.ActorRef;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.LinkedHashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.stream.Collectors;
+import org.apache.commons.collections.CollectionUtils;
+import org.apache.commons.collections.MapUtils;
+import org.apache.commons.lang3.StringUtils;
+import org.sunbird.actorutil.location.impl.LocationClientImpl;
+import org.sunbird.common.exception.ProjectCommonException;
+import org.sunbird.common.models.util.GeoLocationJsonKey;
+import org.sunbird.common.models.util.JsonKey;
+import org.sunbird.common.models.util.ProjectUtil;
+import org.sunbird.common.responsecode.ResponseCode;
+import org.sunbird.models.location.Location;
+
+/**
+ * This class will provide methods to get validated location ids from Location code.
+ *
+ * @author Amit Kumar
+ */
+public class LocationUtil {
+
+  private LocationClient locationClient = new LocationClientImpl();
+  private ObjectMapper mapper = new ObjectMapper();
+
+  /**
+   * This method will validate the list of location code whether its valid or not. If valid will
+   * return the locationId List.
+   *
+   * @param List of location code.
+   * @param Actor reference.
+   * @return List of location id.
+   */
+  public List<String> getValidatedLocationIds(List<String> codeList, ActorRef actorRef) {
+    Set<String> locationIds = null;
+    List<Map<String, Object>> responseLocList = null;
+    responseLocList = locationClient.getLocationsByCodes(actorRef, codeList);
+    List<Location> locationList =
+        responseLocList
+            .stream()
+            .map(s -> mapper.convertValue(s, Location.class))
+            .collect(Collectors.toList());
+    List<String> locationIdList = new ArrayList<>();
+    if (!CollectionUtils.isEmpty(locationList)) {
+      locationIds = getValidatedLocationSet(locationList, actorRef);
+    } else {
+      throw new ProjectCommonException(
```

Comment:
Not necessary to throw this exception here. Taken care in location client call above.